src/workloads/query/TimeSeriesLastpoint.yml

```diff
@@ -53,369 +57,330 @@ LastpointQueryMetaAscTimeAsc: &LastpointQueryMetaAscTimeAsc
       GroupPattern: {_id: "$metadata.sensorId", timestamp: {$last: "$timestamp"}, temp: {$last: "$temp"}}
       GroupPatternTopOrBottom: {$bottom: {sortBy: *sortPatternMetaAscTimeAsc, output: *output}}
 
+LastpointQueryMetaAscTimeDescRepeat100: &LastpointQueryMetaAscTimeDescRepeat100
```

Comment:
These are same as the above exception repeat = 100. Repeat 100 is necessary to collect on-cpu profiling data. Without it, on-cpu profiling data is not collected for the aggregates.